integrations/visual-tagger/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@segment/analytics.js-integration-visual-tagger",
+  "version": "0.1.0",
```

Comment:
This is behind the latest published version https://www.npmjs.com/package/@segment/analytics.js-integration-visual-tagger

Reply:
My bad not catching this in the review. 🤚